.github/workflows/tests.yaml

```diff
@@ -84,6 +88,11 @@ jobs:
               run: |
                   ifconfig -a
                   scripts/tests/test_suites.sh
+            - name: Run TV Tests
+              timeout-minutes: 10
+              run: |
+                  ifconfig -a
```

Comment:
Why do we need the `ifconfig -a` here?  I don't think we do (nor in the clause right above....)

Reply:
Removed.